lib/ProMotion/screen/screen_module.rb

```diff
@@ -166,6 +166,10 @@ def frame
       return self.view_or_self.frame
     end
 
+    def try(method, *args)
+      send(method, *args) if respond_to?(method)
+    end
```

Comment:
Should we consider putting useful methods like this and `app` in their own module so they can easily be included in non-ProMotion classes? I suppose we could even consider monkey patching object like Rails does. :astonished: